pkg/server/commands/connected_objects.go

```diff
@@ -92,18 +92,30 @@ type UserRef struct {
 }
 
 type ConnectedObjectsCommand struct {
-	Datastore storage.OpenFGADatastore
-
+	Datastore        storage.RelationshipTupleReader
+	Typesystem       *typesystem.TypeSystem
```

Comment:
this line isn't needed - you're never reading its contents

Reply:
The Typesystem does not belong in the request 🤦 The Typesystem belongs as part of the scope of the command struct.